pkg/util/namespace/namespace.go

```diff
@@ -3,23 +3,73 @@ package namespace
+		"openshift-cloud-controller-manager":               {},
+		"openshift-cloud-controller-manager-operator":      {},
+		"openshift-cloud-credential-operator":              {},
+		"openshift-cluster-csi-drivers":                    {},
+		"openshift-cluster-machine-approver":               {},
+		"openshift-cluster-node-tuning-operator":           {},
+		"openshift-cluster-samples-operator":               {},
+		"openshift-cluster-storage-operator":               {},
+		"openshift-config":                                 {},
+		"openshift-config-managed":                         {},
+		"openshift-config-operator":                        {},
+		"openshift-console":                                {},
+		"openshift-console-operator":                       {},
+		"openshift-console-user-settings":                  {},
+		"openshift-controller-manager":                     {},
+		"openshift-controller-manager-operator":            {},
+		"openshift-dns":                                    {},
+		"openshift-dns-operator":                           {},
+		"openshift-etcd":                                   {},
+		"openshift-etcd-operator":                          {},
+		"openshift-host-network":                           {},
+		"openshift-image-registry":                         {},
+		"openshift-ingress":                                {},
+		"openshift-ingress-canary":                         {},
+		"openshift-ingress-operator":                       {},
+		"openshift-insights":                               {},
+		"openshift-kni-infra":                              {},
+		"openshift-kube-apiserver":                         {},
+		"openshift-kube-apiserver-operator":                {},
+		"openshift-kube-controller-manager":                {},
+		"openshift-kube-controller-manager-operator":       {},
+		"openshift-kube-scheduler":                         {},
+		"openshift-kube-scheduler-operator":                {},
+		"openshift-kube-storage-version-migrator":          {},
+		"openshift-kube-storage-version-migrator-operator": {},
+		"openshift-machine-api":                            {},
+		"openshift-machine-config-operator":                {},
+		"openshift-marketplace":                            {},
+		"openshift-monitoring":                             {},
+		"openshift-multus":                                 {},
+		"openshift-network-diagnostics":                    {},
+		"openshift-network-operator":                       {},
+		"openshift-oauth-apiserver":                        {},
+		"openshift-openstack-infra":                        {},
+		"openshift-operator-lifecycle-manager":             {},
+		"openshift-operators":                              {},
+		"openshift-ovirt-infra":                            {},
+		"openshift-sdn":                                    {},
+		"openshift-service-ca":                             {},
+		"openshift-service-ca-operator":                    {},
```

Comment:
Will this list of namespaces potentially change over time? I'm just cautious of using this hard-coded list and having to manage it over time

Reply:
@ellis-johnson - Thanks for reviewing. Yes, as we discussed in the ADO task and this PR conversation, we would like to have a more precise filtering in the simplest way possible and the list of the hard-coded namespaces in `pkg/util/namespace/namespace.go` needs to be updated regularly as every minor version of upstream OCP introduces a new namespace or two. 

In the next iteration, we probably will do a more intelligent way to extract namespaces from an OCP release (similar to what OSD is doing [here](https://github.com/openshift/managed-cluster-config/blob/1d5397c516289da8f3ef4fc4c8ab306566ba8b8a/scripts/managed-resources/generate-managed-list.py#L44-L75)) and mix them with ARO-specific namespaces.